code/nodes/saveGraph/saveGraph.q

```diff
@@ -1,6 +1,24 @@
-// Save all the graphs relevant for the generation of reports and for prosperity
 \d .automl
 
+// Save all the graphs relevant for the generation of reports and for prosperity
+
+// @kind function
+// @category node
+// @fileoverview Save all graphs needed for reports 
+// @param params {dict} All data generated during the preprocessing and
+//  prediction stages
+// @return {null} All graphs needed for reports are saved to appropriate location
+saveGraph.node.function:{[params]
+  savePath:params[`pathDict;`images];
+  saveGraph.targetPlot[params;savePath];
+  saveGraph.confusionMatrix[params;savePath];
```

Comment:
Can probably add in an if/else statement to the saveGraph here to check for problem type, this is useful incase we want to add more graphs in down the line and they wouldn't need to be checked internally to multiple function